modules/featurecounts.nf

```diff
@@ -3,26 +3,26 @@
 * TODO: check that we do not miss a gene due to the 'sed 1d' remove of the first two lines
 ************************************************************************/
 process featurecounts {
-  label 'subread'
-  publishDir "${params.output}/${params.dir}", mode: 'copy', pattern: "${name}.counts*"
+    label 'subread'
+    publishDir "${params.output}/${params.featurecounts_dir}", mode: 'copy', pattern: "${name}.counts*"
 
-  input:
-  tuple val(name), file(bam)
-  file(annotation)
+    input:
+    tuple val(name), path(bam)
+    path(annotation)
 
-  output:
-  tuple val(name), file("${name}.counts"), emit: counts // [mock_rep1, /home/hoelzer/git/nanozoo/wf_gene_expression/work/9e/7fb58903c9e4163d526ef749c0d088/mock_rep1.counts]
-  path "${name}.counts.summary", emit: log
+    output:
+    tuple val(name), path("${name}.counts"), emit: counts // [mock_rep1, /home/hoelzer/git/nanozoo/wf_gene_expression/work/9e/7fb58903c9e4163d526ef749c0d088/mock_rep1.counts]
```

Comment:
``emit:`` gives the output channel a name, right?

Reply:
yes, indeed. 
you can access the output of a process in the workflow with process_name.out.output_name